docs/deploy-with-docker.md

```diff
@@ -38,6 +38,10 @@ $ docker build -t apisix-dashboard:$tag .
 
 # For users in mainland China, the `ENABLE_PROXY` parameter can be provided to speed up module downloads.
 $ docker build -t apisix-dashboard:$tag . --build-arg ENABLE_PROXY=true
+
+# If you want to use the latest codes to build, you can specify the `APISIX_DASHBOARD_VERSION` parameter to `master`. 
+# This parameter can also be specified as branch name of a specific version, such as `v2.0`.
+$ docker build -t apisix-dashboard:$tag . --build-arg APISIX_DASHBOARD_VERSION=master
```

Comment:
what is the default value of `APISIX_DASHBOARD_VERSION`?

Reply:
now it's v2.0. maybe master is better ?

Reply:
I think it should be 2.1 now. 

APISIX has release 2.1 today ^_^

Reply:
no, use 2.1 is because the dashboard has released the 2.1.

Reply:
@juzhiyuan you are right